invokeai/backend/model_management/convert_ckpt_to_diffusers.py

```diff
@@ -1026,6 +1026,15 @@ def convert_open_clip_checkpoint(checkpoint):
 
     return text_model
 
+def replace_checkpoint_vae(checkpoint, vae_path:str):
+    if vae_path.endswith(".safetensors"):
+        vae_ckpt = load_file(vae_path)
+    else:
+        vae_ckpt = torch.load(vae_path, map_location="cpu")
```

Comment:
Do we need to look into safety here since (I think) torch saves/restores pickled files?

Also, what if the load (either method) fails?

Reply:
Please see PR #3011 for a fix for this exact issue.